rocAL/include/readers/image/image_reader.h

```diff
@@ -171,6 +171,10 @@ class Reader {
     //! Returns the name/identifier of the last item opened in this resource
     virtual std::string id() = 0;
     //! Returns the number of items remained in this resource
+
+     //! Returns the path of the last item opened in this resource
+    virtual std::string file_path() { return {}; }
```

Comment:
why is this returning an empty string?

Reply:
Currently we are using it in the file_source_reader.cpp and not in other readers, in order to avoid adding a definition in other readers which would increase file changes we have set it as a virtual function returning an empty string